src/server.rs

```diff
@@ -220,6 +227,25 @@ impl Server {
             });
         });
 
+        let calls_count = self.calls_count.clone();
+        Python::with_gil(|py| {
+            py.allow_threads(|| {
+                thread::spawn(move || loop {
+                    let ttl = SystemTime::now()
+                        .duration_since(UNIX_EPOCH)
+                        .unwrap()
+                        .as_secs();
```

Comment:
Correct me if I am wrong but the `ttl` variable here is the seconds elapsed from `1970-1-1`?

Reply:
Then after 60 seconds(thread::sleep) , this variable will be t+60, and remove all the previous timestamps, right?

Reply:
I am wondering if the `ttl` specified in the RateLimit constructor and the `ttl` variable here should be connected 🤔

Reply:
Yes, this TTL is for retention of old calls, the idea for this thread is to clean up calls of old users, to avoid having the calls_count inflate endlessly. Not sure it should be related to the TTL for the RateLimiter, especially since different endpoints (RateLimiter instances) can have different TTLs.

Reply:
@IdoKendo , I see, this makes sense. Maybe in the future we could also add the ability to clear the cache based on the size of the cache.

But one thing - let us have an environment variable to make it configurable.